app/Plugin/CoreApi/Model/CoreApi.php

```diff
@@ -26,18 +26,36 @@
  */
 
 App::uses("AppModel", "Model");
+App::uses('Utility', 'Validation');
 
 class CoreApi extends AppModel {
   // Define class name for cake
   public $name = "CoreApi";
 
   // Required by COmanage Plugins
-// XXX could have an "api" plugin type though there's not much value add for it atm
+  // XXX could have an "api" plugin type though there's not much value add for it atm
   public $cmPluginType = "other";
   
   // Add behaviors
   public $actsAs = array('Changelog', 'Containable');
-  
+
+  // List of allowed query parameters
+  // 1. range, inList, comparison are all validation rules supported by CAKE core
+  //    In runtime it will be translated to Validation::validation_rule, Validation::email
+  // 2. CAKEPHP 2.x uses lowercased and underscored URLS (InflectedRoute class in CAKEPHP 3.0)
+  //    as a result person.status becomes person_status
+  const ALLOWED_QUERY_PARAMS = array(
```

Comment:
Is there a reason to define this as `const` rather than a private/protected var?

Reply:
No. Will change.